docs/GETTING_STARTED.rst

```diff
@@ -1,9 +1,8 @@
 Getting Started
 ---------------
 
+- `Overview of TUF <OVERVIEW.rst>`_
 - `Installation <INSTALLATION.rst>`_
-- `Contributors <CONTRIBUTORS.rst>`_
-- `Quickstart <QUICKSTART.md>`_
-- `CLI <CLI.md>`_
-- `CLI Usage Examples <CLI_EXAMPLES.md>`_
-- `Tutorial <TUTORIAL.md>`_
+- Beginner Tutorials (using the basic command-line interface): `Quickstart <QUICKSTART.md>`_; `CLI Tutorial <CLI.md>`_; `CLI Further Examples <CLI_EXAMPLES.md>`_
```

Comment:
The long line looks a little daunting now. I would leave QUICKSTART.md separate and see if the cli tutorials could be combined, or restructured? From a quick glance, CLI.md looks like a documentation of the interface, whereas CLI_EXAMPLES.md looks like a slightly more in-depth version of QUICKSTART.md.

Reply:
Fixed, and I've now combined the two CLI documents.